### PR TITLE
chore(updatecli):Track `ubuntu 22_04 AMD64` AZ images

### PR DESCRIPTION
**Pull request by jayfranco999**
As per https://github.com/jenkins-infra/helpdesk/issues/4365#issue-2610776824

This PR pins and tracks the latest AZ image for `ubuntu 22_04 AMD64`

### CODE REVIEW

File: updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    name: Update azure `ubuntu 22.04` amd64 image version in locals
+    sourceid: lastReleaseVersion
+    kind: yaml
+    scmid: default
+    spec:
+      file: ./images-versions.yaml
+      key: $.azure.ubuntu.'22.04'.amd64
+    transformers:
+      - addprefix: '"'
+      - addsuffix: '"'
```
Comment (dduportal) [inline updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml:39]: Did you test locally? If yes what was the result for your test?

On the CI side, it fails with the following error:

```
ERROR: something went wrong in "target#updateVersion" : something went wrong in target "updateVersion" : "validation error: transformer input is empty"
```

**Review by dduportal — CHANGES_REQUESTED** [review]
Good work! Still some work is needed:

- Blocker: fix the `updatecli` errors. It looks like the source responds with an empty string on the CI. Not sure if it is an updatecli error, AZ credential error, or something else.
- Rebase the PR against the latest main reference

**Comment by jayfranco999** [comment]
Update:

The az command to fetch the latest `AZ Ubuntu 22_04 AMD64` image now works on infra.ci (https://infra.ci.jenkins.io/job/updatecli/job/packer-images/job/PR-1507/32/pipeline-console/log?nodeId=18)

```
[2024-11-26T10:11:51.136Z] #################################################
[2024-11-26T10:11:51.136Z] # BUMP AZURE `UBUNTU 22.04` AMD64 IMAGE VERSION #
[2024-11-26T10:11:51.136Z] #################################################
[2024-11-26T10:11:51.136Z] 
[2024-11-26T10:11:51.136Z] source: source#lastReleaseVersion
[2024-11-26T10:11:51.136Z] -------------------------
[2024-11-26T10:12:23.134Z] The shell 🐚 command "/bin/sh /tmp/updatecli/bin/fe287e953d07bf4def15ea4f2d79cd182990dee39f04530a41f3a8f0e0944d01.sh" ran successfully with the following output:
[2024-11-26T10:12:23.134Z] ----
[2024-11-26T10:12:23.134Z] 22.04.202410020
[2024-11-26T10:12:23.134Z] ----
[2024-11-26T10:12:23.134Z] ✔ shell command executed successfully
[2024-11-26T10:12:23.134Z] 
[2024-11-26T10:12:23.134Z] target: target#updateVersion
[2024-11-26T10:12:23.134Z] --------------------
[2024-11-26T10:12:23.134Z] [transformers]
[2024-11-26T10:12:23.134Z] ✔ Result correctly transformed from "22.04.202410020" to "\"22.04.202410020"
[2024-11-26T10:12:23.134Z] ✔ Result correctly transformed from "\"22.04.202410020" to "\"22.04.202410020\""
[2024-11-26T10:12:23.134Z] 
[2024-11-26T10:12:23.134Z] **Dry Run enabled**
[2024-11-26T10:12:23.134Z] 
[2024-11-26T10:12:23.134Z] ⚠ - change detected:
[2024-11-26T10:12:23.134Z] 	* key "$.azure.ubuntu.'22.04'.amd64" should be updated from "\"latest\"" to "\"22.04.202410020\"", in file "./images-versions.yaml"
[2024-11-26T10:12:23.134Z]

File: updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    spec:
+      title: Bump azure `ubuntu 22.04` amd64 image version
+      description: "Update the latest Azure Ubuntu 22.04 amd64 image version in the images-versions.yaml file."
+      labels:
+        - enhancement
```
Comment (dduportal) [inline updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml:49]:
```suggestion
      labels:
        - enhancement
        - ubuntu
```

File: updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    transformers:
+      - addprefix: '"'
+      - addsuffix: '"'
+
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    spec:
+      title: Bump azure `ubuntu 22.04` amd64 image version
```
Comment (dduportal) [inline updatecli/updatecli.d/az-ubuntu-22-04-amd64-images.yml:46]:
```suggestion
      title: Bump Azure Ubuntu `22.04` amd64 image version
```

**Review by dduportal — APPROVED** [review]
(2 nitpicking suggestions which I'm going to apply and merge the PR)

LGTM! Nice work!

**Pull request merged**